### DISCUSSION TITLE
لماذا

### DISCUSSION THREAD

**Discussion by tewfygmaes** [community/community#113942] [Actions] [answered]
### Select Topic Area

Question

### Body

لماذا لا يتم ارسال اشعارات من إلى البوت من ترون والى جيثب لكي يعطيك امان في حفظ بيانات الدخول على الرابط

**Comment by queenofcorgis** [comment, marked as answer]
Hi @tewfygmaes , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
